docs/migration/migrate_7_7.asciidoc

```diff
@@ -62,35 +67,26 @@ const myPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyPluginAp
 );
 ```
 
-For more information on where to locate new platform `data` services, please refer to the table of [plugins for shared application services](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#plugins-for-shared-application-services) in `src/core/MIGRATION.md`.
-
-*via [#60449](https://github.com/elastic/kibana/pull/60449)*
+For more information on where to locate new platform `data` services,
+please refer to the table of
+https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#plugins-for-shared-application-services[plugins for shared application services]
+in `src/core/MIGRATION.md`.
 
-## Delete FilterStateManager and QueryFilter :-D
+*via {pull}60449[#60449]*
 
-Deleted unused legacy exports
- - Delete `FilterStateManager`  
- - Delete `QueryFilter`
- - Delete `SavedQuery` re-export from legacy
+[discrete]
+==== Delete FilterStateManager and QueryFilter
 
-### Checklist
+Deleted unused legacy exports:
 
-Delete any items that are not applicable to this PR.
+* Delete `FilterStateManager`
+* Delete `QueryFilter`
+* Delete `SavedQuery` re-export from legacy
```

Comment:
Change the section to "Delete  unused legacy exports `FilterStateManager`, `QueryFilter`, and `SavedQuery`"